modules/RTC/JitsiLocalTrack.js

```diff
@@ -340,7 +340,7 @@ JitsiLocalTrack.prototype._setMute = function (mute) {
             return self._sendMuteStatus(mute);
         })
         .then(function() {
-            self.eventEmitter.emit(JitsiTrackEvents.TRACK_MUTE_CHANGED);
+            self.eventEmitter.emit(JitsiTrackEvents.TRACK_MUTE_CHANGED, this);
```

Comment:
Why do you need this parameter? You have to attach the listener to the track and this means that the listener knows which is the track.